src/extension.ts

```diff
@@ -524,3 +527,7 @@ export function applyTextEdit(we) {
         vscode.workspace.applyEdit(wse)
     }
 }
+
+export function lintPackage() {
+    languageClient.sendRequest("julia/lint-package")
+}
```

Comment:
I don't understand this, maybe push an empty commit and see if that clears this up